acceptance/suites/tests/service_framework/reload_updated_crl.rb

```diff
@@ -25,8 +30,12 @@
         "then mv -f #{inventory_file}.bak #{inventory_file}; fi")
   end
 
+  step 'Disable crl reloading in bootstrap.cfg' do
+    on(master, "sed -i 's,#{watcher_service},##{watcher_service},' #{bootstrap}")
```

Comment:
From what I can see it doesn't sound like a concern, maybe I'm missing something